tests/x11/rrdtool_x11.pm

```diff
@@ -0,0 +1,87 @@
+# SUSE"s openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Regression test for rrdtool
+# - created rrd file
+# - update rrd file
+# - fetch data from file
+# - generate images with different options
+# - compare images with needles
+#
+# Expected result:
+# 4 graphics will be displayed without error.
+#
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+my $imagespeed = "";
+
+# open images and check needles.
+sub open_image {
+    send_key("alt-f2");
+    type_string("eog /tmp/rrdtool/$imagespeed.png\n");
+    sleep 5;
```

Comment:
You don't need sleep before assert_screen.

Reply:
ok, I really have a doubt about this, removing and works fine.